Event/include/Event/TrigScintCluster.h

```diff
@@ -0,0 +1,191 @@
+/**
+ * @file TrigScintCluster.h
+ * @brief Class that stores cluster information from the TrigScint 
+ * @author Josh Hiltbrand, University of Minnesota
+ * @author Lene Kristian Bryngemark, Stanford University : adaptation to TrigScint 
+ */
+
+#ifndef EVENT_TRIGSCINTCLUSTER_H_
+#define EVENT_TRIGSCINTCLUSTER_H_
+
+// ROOT
+#include "TObject.h"
+#include "TString.h"
+#include "TRefArray.h"
```

Comment:
Please remove TString.h and TRefArray.h includes. I don't think you use them here.